### PR TITLE
Improvements

### PR DESCRIPTION
**Pull request by hypeJunction**
anypage-list
anypage

### CODE REVIEW

File: classes/AnyPage.php [hunk 1]
```diff
@@ -32,8 +44,8 @@ public function save() {
 		$site = elgg_get_site_entity();
 
 		$this->access_id = ACCESS_PUBLIC;
-		$this->container_guid = $site->getGUID();
-		$this->owner_guid = $site->getGUID();
+		$this->container_guid = $site->guid;
+		$this->owner_guid = $site->guid;
```
Comment (brettp) [inline classes/AnyPage.php]: Curious why this change was needed?

Comment (hypeJunction) [inline classes/AnyPage.php]: We have decided between ourselves that `->guid` is preferred by core for better readability, plus it saves a method call, reduces the risk of fatal errors.

Comment (brettp) [inline classes/AnyPage.php]: I don't agree with these points, especially about the "benefit" of avoiding fatal errors (you don't want to call properties on the wrong objects any more than calling methods on the wrong objects), but it's fine here.

Comment (hypeJunction) [inline classes/AnyPage.php]: We've discussed this quite extensively somewhere. Too late now.

Comment (brettp) [inline classes/AnyPage.php]: Where? It should never be too late to improve practices, and definitely not for other projects.
